blockly/apps/blocklyduino/index.html

```diff
@@ -46,6 +46,9 @@
     }
   }
 
+  if (document.getElementById('tab_blocks').className == 'tabon') {
+    Blockly.mainWorkspace.setVisible(false);
+  }
```

Comment:
This and the opposite from line 66 are required to fully hide the workspace on tab click. Otherwise the toolbox stays on top of the other tab content.